AppInsightsCommon/src/Telemetry/Common/DataSanitizer.ts

```diff
@@ -179,4 +178,12 @@ export class DataSanitizer {
         var s = "00" + num;
         return s.substr(s.length - 3);
     }
+
+    /**
+     * helper method to trim strings (IE8 does not implement String.prototype.trim)
+     */
+    public static trim(str: any): string {
```

Comment:
datasanitizer in common has it, maybe later expose it.

Reply:
This is just a hack to remove common's circular dependency. Can look into a better solution in the future